src/@orb.yml

```diff
@@ -1,148 +1,9 @@
-                executor: my-custom-executor
-
-  add-git-and-ssh:
-    description: |
-      Pre-install git and ssh, for example to run tests against a git tag.
-    usage:
-      version: 2.1
-
-      orbs:
-        shellcheck: circleci/shellcheck@x.y.z
-
-      workflows:
-        shellcheck:
-          jobs:
-            - shellcheck/check:
-                pre-steps:
-                  - run:
-                      name: "Install git and SSH client"
-                      command: |
-                        apk add \
-                            --update-cache \
-                            --no-progress \
-                            git \
-                            openssh-client
-jobs:
-  check:
-    description: |
-      Run shellcheck over any shell scripts in the respository.
-      Add this job to any workflow to automatically check any shell scripts
-      in your repository.
-    executor: << parameters.executor >>
-    parameters:
-      exclude:
-        type: string
-        default: ""
-        description: |
-          This file pattern (as passed to `find -not -path`) is used to select
-          a path to exclude when searching for files. Currently, this is
-          limited to a single pattern.
-      ignore:
-        type: string
-        default: ""
-        description: >
-          Comma-separated string list of error types to ignore, e.g.,
-          `SC2059,SC2034,SC1090`
-      path:
-        type: string
-        default: .
-        description: The top-most directory to search for files in.
-      pattern:
-        type: string
-        default: "*.sh"
-        description: |
-          The file pattern as passed to `find` to locate shell scripts.
-      output:
-        type: string
-        default: shellcheck.log
-        description: |
-          The filename of the shellcheck output, which is automatically saved
-          as a job artifact.
-      executor:
-        type: executor
-        default: shellcheck
-        description: |
-          Override executor with a custom executor, for example, to choose a
-          different version tag.
-    steps:
-      - checkout
-      - run:
-          name: Check Scripts
-          command: |
-            <<#parameters.ignore>>
-            IGNORE_STRING=<<parameters.ignore>>
-            SHELLCHECK_OPTS=$(echo "-e ${IGNORE_STRING//,/ -e }")
-            export SHELLCHECK_OPTS="$SHELLCHECK_OPTS"
-            echo "Running shellcheck with the following SHELLCHECK_OPTS value..."
-            echo "SHELLCHECK_OPTS=$SHELLCHECK_OPTS"
-            <</parameters.ignore>>
-            find '<<parameters.path>>' -not -path '<<parameters.exclude>>' \
-              -type f -name '<<parameters.pattern>>' | \
-              xargs shellcheck --external-sources | \
-              tee -a '<<parameters.output>>'
-      - store_artifacts:
-          path: '<< parameters.output >>'
-          destination: shellcheck
+  homepage: https://shellcheck.net
```

Comment:
I believe this should be `home_url`, to be properly rendered in the orb registry